pulsar-common/src/test/java/org/apache/pulsar/common/nar/NarUnpackerTest.java

```diff
@@ -157,4 +160,17 @@ File findJavaExe() {
         File javaExe = new File(javaHome, "bin/java" + (SystemUtils.IS_OS_WINDOWS ? ".exe" : ""));
         return javaExe;
     }
-}
\ No newline at end of file
+    @Test
+    public void testForPermissionSet() throws IOException{
+        File file = File.createTempFile("TestFilePerm", "test");
+	file.delete();
```

Comment:
please use space instead of tab.
```suggestion
        file.delete();
```

Reply:
Thanks @Jason918 ,I fix this problem in this commit. Could you please have a look at this?